ext/net/ops.rs

```diff
@@ -665,6 +666,26 @@ where
   Ok(results)
 }
 
+pub async fn op_set_nodelay<NP>(
+  state: Rc<RefCell<OpState>>,
+  args: NoDelayArgs,
+  _: (),
+) -> Result<(), AnyError> {
+  let rid = args.rid;
+  let nodelay = args.nodelay;
+  let resource: Rc<TcpStreamResource> = state
+    .borrow_mut()
+    .resource_table
+    .get::<TcpStreamResource>(rid)?;
+  resource.set_nodelay(nodelay).await
+}
+
+#[derive(Deserialize)]
+pub struct NoDelayArgs {
+  rid: ResourceId,
+  nodelay: bool,
+}
+
```

Comment:
This struct is no longer necessary if we flatten op args (which is preferred if possible)

```suggestion
```

Reply:
Good suggestion. Thanks!